wikiHandler.js

```diff
@@ -18,7 +21,8 @@ const ALL_ITEM_KEYS = new Set([
     ...Object.keys(SPELLS),
     ...Object.keys(ALIASES),
 ]);
-const SERVER_COLOR = {BLUE: '#1e1e92', GREEN: '#008000'};
+const SERVER_COLOR = {BLUE: '#1C58B8', GREEN: '#2FC374'};
```

Comment:
Sorry for the bikeshedding, but... I liked the old colors :(
RIP. What was the motivation for changing these?

Reply:
I needed a third color for system related messages like !list and !help and didn't like any of the hues that went with the original server colors.  At the same time, I get several users who claim their watch notifications aren't coming through because they saw a tunnel stream post on the other server for their item.  I wanted to see if making the colors brighter would make it more obvious which server they are looking at.  Also just think these look slicker, but that's just my opinion.  Can put it up as a poll and see what users want.

Reply:
Ah, I thought maybe the green change was a reference to Teal, since it was so light. 🤣 
I wouldn't waste too much time on it, it's really just bikeshedding. Was just curious.
I think the "other server confusion" thing happens when people use search and Discord doesn't automatically add the `in:#p99_green_tunnel_stream` filter for some reason. I've talked to a couple of people who were confused because of this. Not sure if we can fix this easily...

Reply:
I made Green's color a bit darker